Yoast/yoast-components
# Add classnames to dashboard widget

## Summary

This PR can be summarized in the following changelog entry:

* Adds class names to `WordpressFeed` and `SeoAssessment` components, and also to their children.

## Test instructions

This PR can be tested by following these steps:

* Run the development environment and navigate to the Dashboard widget tab
* Inspect the elements to see the added classnames
